inspirehep/modules/workflows/tasks/arxiv.py

```diff
@@ -71,7 +74,15 @@ def arxiv_fulltext_download(obj, eng):
     )
 
     if pdf:
-        obj.log.info('PDF retrieved from arXiv for %s', arxiv_id)
+        if pdf.get_version().mimetype == 'application/pdf':
+            obj.log.info('PDF retrieved from arXiv for %s', arxiv_id)
+        else:
+            # We need to delete the failed PDF
+            del obj['files'][filename]
+            pdf.delete()
+            obj.log.warning('PDF retrieved from arXiv for %s', arxiv_id)
+            raise RuntimeError("PDF from arXiv for {arxiv_id} isn't yet ready".format(
```

Comment:
> BTW I think that this kind of exception message will generate one distinct unclustered message on Sentry per failure.

But it's not possible to aggregate per exception, can I? I can aggregate log messages, but what about exceptions?